lib/engine/config/game/g_1860.rb

```diff
@@ -576,35 +576,35 @@ module G1860
   "trains": [
     {
       "name": "2+1",
-      "distance": 2,
+      "distance": [2, 1],
```

Comment:
why is distance an array?

look at the format of distance in g_1846

Reply:
how can your code work with distance in this format?

Reply:
M+N train => distance = [M, N]
Since I was rewriting the code dealing with distances, I chose to use a simple array over the more complicated hash used by 1846 and the others. In addition, the existing code and the hash structure couldn't handle how 1860 train distances work - specifically how an unused city/offboard allowance can be used for towns/halts (i.e. a run than doesn't use all M cities).

Reply:
i don't think that's true. the current hash used in 1846 should allow for city and offboard overflowing to towns and halts.

Reply:
OK, I'll look at the code again. I wasn't seeing how that could work.

Reply:
Done.